qiskit/pulse/schedule.py

```diff
@@ -249,6 +250,60 @@ def filter(self, *filter_funcs: List[Callable],
             time_ranges: Time intervals to keep, e.g. [(0, 5), (6, 10)]
             intervals: Time intervals to keep, e.g. [Interval(0, 5), Interval(6, 10)]
         """
+        total_filter = self._construct_filter(*filter_funcs,
+                                              channels=channels,
+                                              instruction_types=instruction_types,
+                                              time_ranges=time_ranges,
+                                              intervals=intervals)
+        subschedules = self.flatten()._children
+        valid_subschedules = [sched for sched in subschedules if total_filter(sched)]
+        return Schedule(*valid_subschedules, name="{name}-filtered".format(name=self.name))
+
+    def exclude(self, *filter_funcs: List[Callable],
+                channels: Optional[Iterable[Channel]] = None,
+                instruction_types: Optional[Iterable[Type['Instruction']]] = None,
+                time_ranges: Optional[Iterable[Tuple[int, int]]] = None,
+                intervals: Optional[Iterable[Interval]] = None) -> 'Schedule':
+        """
+        Return a Schedule with only the instructions *failing* at least one of the provided filters.
+        That is, this function has the same inputs as self.filter, but returns the complement
+        schedule, so that self.filter(args) | self.exclude(args) == self
```

Comment:
Document args. The arg documentation for these three methods should all be the same